en_US/getting-started/install.md

```diff
@@ -622,7 +622,7 @@ $ cd emqx-rel && make
 4. Start EMQ X Broker
 
 ```bash
-$ cd _build/emqx-rel/_rel/emqx
+$ cd _build/emqx/rel/emqx
 
 $ ./bin/emqx start
 emqx 4.0.0 is started successfully!
```

Comment:
we should probably use a 4.3-beta.1 version as an example here

Reply:
the same at line 632.

Reply:
This is the 4.2 version of the document, so the previous compilation method will be used, and the 4.3 document will be updated later